frontend/src/scenes/dashboard/NewDashboardModal.scss

```diff
@@ -0,0 +1,8 @@
+.DashboardTemplateChooser {
+    max-width: 780px;
+}
+
+.TemplateItem {
+    width: 240px;
+    height: 210px;
+}
```

Comment:
These CSS classes will now be applied globally, and thus whenever any other page accidentally internally uses `TemplateItem` as a class name, it'll be in a weird boxy shape.

I suggest nesting them around a `.NewDashobardModal { /* here */ }` container.